### ISSUE TITLE
Broken asset links when extending layout from nested folder

### ISSUE THREAD

**Issue by robertjwhitney** [carrot/roots-css-pipeline#7]
TL;DR The js, and css pipelines default "out" directory is a relative path to the current directory rather than relative to the root url (needs to have a leading `/`). Took a look, couldn't decide where/how best to do this.

## Steps to reproduce

In a new roots project

- Create a jade template nested in a subfolder like `/some/sub/category/index.jade`
- Extend the main layout file in that template
- css and js attempt to load from `/some/sub/category/build.css`, fails.

## Resolving the issue from app

Using "out" option to specify path relative to domain rather than directory:

```
extensions: [
    js_pipeline(files: 'assets/js/*.coffee', out: '/js/main.js'),
    css_pipeline(files: 'assets/css/*.styl', out: '/css/master.css')
  ]
```

## Resolving the issue from this project

Prepend a leading `/` to the default "out" path.

**Comment by robertjwhitney** [comment]
Oh crap- I see a commit here that you do this on purpose? Ok.

**Comment by robertjwhitney** [comment]
RTFM I got it now

**Comment by kylemac** [comment]
:heart:

**Comment by jenius** [comment]
:cactus:

**Comment by robertjwhitney** [comment]
:grin:
